website/docs/migration/diff.md

```diff
@@ -1,11 +1,18 @@
 ---
 sidebar_position: 11
+slug: /migration/diff/ # referenced in README.md and redirects
+description: Supported commands
 ---
 
 # Known differences
 
 We don't plan to address those known differences in behavior:
```

Comment:
Could we put this sentence sub section or rephrase?  "# Known differences" is header 1 and "## Administrative commands" is header 2, which makes it read to me that, we don't plan to address _those_ known differences, indicating _those_ as differences in list of commands below.